Gems/Terrain/Code/Source/TerrainRenderer/TerrainFeatureProcessor.cpp

```diff
@@ -341,6 +340,8 @@ namespace Terrain
                 uint16_t bottomLeft = (y + 1) * gridVertices + x;
                 uint16_t bottomRight = bottomLeft + 1;
 
+                constexpr uint16_t one = 1;
+
```

Comment:
Should this be removed?

Reply:
Ugh I had a really messy rebase happen at one point and this probably slipped through. Will fix.